src/main/java/uk/gov/justice/hmpps/prison/service/PrisonerSearchService.kt

```diff
@@ -48,7 +48,7 @@ class PrisonerSearchService(
           inOutStatus = it.inOutStatus,
           identifiers = it.identifiers?.sortedBy { it.whenCreated },
           sentenceDetail = it.sentenceDetail?.apply { additionalDaysAwarded = booking?.additionalDaysAwarded ?: 0 },
-          mostSeriousOffence = it.offenceHistory?.filter { off -> off.bookingId == it.bookingId }?.firstOrNull { it.mostSerious }?.offenceDescription,
+          mostSeriousOffence = it.offenceHistory?.filter { off -> off.bookingId == it.bookingId }?.filter { it.mostSerious }?.minByOrNull { it.offenceSeverityRanking }?.offenceDescription,
```

Comment:
There are some prisoners in NOMIS with multiple most serious offences on the same booking. In that case we'll pick the lowest `OFFENCE_SEVERITY_RANKING`.